src/main/java/eu/xenit/transformer/arender/alfresco/content/transformer/ContentTransformARenderRestClient.java

```diff
@@ -19,33 +19,28 @@
 
 public class ContentTransformARenderRestClient {
 
-    private static final Logger LOGGER = Logger.getLogger(ContentTransformARenderRestClient.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(ContentTransformARenderRestClient.class);
 
     private String address;
 
     private final RestTemplate template;
 
     private final SimpleClientHttpRequestFactory clientHttpRequestFactory;
 
-    // upgrade transfer partial size, 8MB at a time
-    private static final int partialTransferBlockSize = 8 << 20;
-
     public ContentTransformARenderRestClient() {
         template = new RestTemplate();
         clientHttpRequestFactory = new SimpleClientHttpRequestFactory();
         clientHttpRequestFactory.setConnectTimeout(120000);
         clientHttpRequestFactory.setReadTimeout(120000);
         template.setRequestFactory(clientHttpRequestFactory);
-
     }
 
     public float getWeatherPerformance() {
         try {
             return template.getForObject(UriComponentsBuilder.fromHttpUrl(address + "weather").build().toUri(),
                     Float.class);
         } catch (Exception e) {
-            LOGGER.log(java.util.logging.Level.INFO,
-                    "Caught exception while trying to fetch weather score, is rendition server down?", e);
+            LOGGER.info("Caught exception while trying to fetch weather score, is rendition server down?", e);
```

Comment:
Shouldn't this be at error level ? Or at least warning ?

Reply:
This is used to determine if the rendition server is available.
If it is not available, errors will be thrown from somewhere else when trying to do a rendition